src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -25,6 +27,13 @@
      */
     public PersonListPanel(ObservableList<Person> personList) {
         super(FXML);
+        //Prevent mouse selection
+        personListView.addEventFilter(MouseEvent.MOUSE_PRESSED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                event.consume();
+            }
+        });
```

Comment:
nitpick: lets keep it consistent with the other UI components and leave a new line here

Reply:
We don't have to. The list cell can still be selected i.e the green vertical bar will still pop out when you type in the command like previously implemented.

Reply:
Ok, thats great!